tpv34/tpv34_fault.yaml

```diff
@@ -0,0 +1,49 @@
+!Switch
+[mu_d, mu_s, d_c]: !ConstantMap
+  map:
+    mu_d: 0.45
+    mu_s: 0.58
+    d_c: 0.18
+[cohesion]: !FunctionMap
+  map:
+    cohesion: |
+      return 0.0-425.0*max(z+2400.0,0.0);
```

Comment:
remove 0-

Reply:
you mean remove 0.0 only? I think the negative is still needed

Reply:
yes